staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -188,12 +186,10 @@ func (q *delayingType) waitingLoop() {
 			return
 		}
 
-		now := q.clock.Now()
```

Comment:
restore this so we have a consistent view of `now` throughout the loop's iteration